eng/Versions.props

```diff
@@ -0,0 +1,8 @@
+<Project>
```

Comment:
Is this file modified with the automatic bump logic too?

Reply:
Yeah, `darc update-dependencies` will update both `Version.Details.xml` and `Versions.props`. Here's an example of what that looks like: https://github.com/xamarin/xamarin-android/pull/5744/files